internal/backup_sentinel_dto.go

```diff
@@ -3,10 +3,13 @@ package internal
 import (
 	"sync"
 	"time"
+
+	"github.com/wal-g/wal-g/utility"
 )
 
 // BackupSentinelDto describes file structure of json sentinel
 type BackupSentinelDto struct {
+	BackupType        string  `json:"Type"`
```

Comment:
Three changes needed:
1. I couldn't find the place where BackupType is being set.
2. We already have the information to determine if backup is full or delta. Let's leave only the "remote"/"local" types.
3. BackupSentinelDto can be quite heavy (> 1GB) because it contains the backup files list. I suggest to move the BackupType to metadata to allow quick check if backup is remote.

Reply:
That is a leftover from another implementation. I cleaned it out.